test/test_ruby_http_client.rb

```diff
@@ -31,9 +31,11 @@ def setup
         ')
     @host = 'http://localhost:4010'
     @version = 'v3'
```

Comment:
@jrectenwald Would you be OK with adding a duplicate test to this one, that doesn't pass http_options, so we can prove in the tests that it works both ways?

Reply:
@mbernier How's that?